src/platform/Tizen/WiFiManager.cpp

```diff
@@ -877,6 +1134,44 @@ CHIP_ERROR WiFiManager::GetSecurityType(wifi_manager_security_type_e * securityT
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR WiFiManager::GetConfiguredNetwork(NetworkCommissioning::Network & network)
+{
+    wifi_manager_ap_h connectedAp = _WiFiGetConnectedAP();
+    size_t length                 = 0;
+    if (connectedAp == nullptr)
+    {
+        return CHIP_ERROR_INCORRECT_STATE;
+    }
+    char * essid = nullptr;
+    int wifiErr  = wifi_manager_ap_get_essid(connectedAp, &essid);
+    VerifyOrReturnError(wifiErr == WIFI_MANAGER_ERROR_NONE, CHIP_ERROR_INTERNAL,
+                        ChipLogError(DeviceLayer, "FAIL: get essid [%s]", get_error_message(wifiErr)));
+    length = sizeof(network.networkID);
+    memcpy(network.networkID, essid, length);
+    if (length >= strlen(essid))
```

Comment:
If this tests true, line 1150 was a buffer overrun.

Reply:
I have changed the order of memcpy and if conditions, so length will be updated first then it will copy only required length of the byte. Thanks for the comment, I got to know difference between overflow and overrun :).
Please check once.